site/js/main.js

```diff
@@ -0,0 +1,418 @@
+// Import all required modules
```

Comment:
Rename file to `main.mjs`?

Reply:
To explicitly mark it as an ES module? I've mainly seen .mjs used in Node.js environments. In this case, the type="module" attribute already makes it somewhat clear that it's an ESM. Is there a specific advantage to using .mjs for browser modules that I'm missing?

Reply:
You are right. The type attribute makes it clear for the browser. But while working with them it could be easier.
But fine keeping the names